### ISSUE TITLE
Regenerate the lock file with the new reth-ipc location

### ISSUE THREAD

**Issue by carver** [ethereum/trin#786]
It moved here: https://github.com/paradigmxyz/reth/tree/main/crates/rpc/ipc

If you rebuild the Cargo.lock file, you'll see an error currently. So let's update to their latest.

**Comment by Perelyn-sama** [comment]
This seems straightforward, I can handle it

**Comment by Perelyn-sama** [comment]
I don't get an error when I rebuild, I'm not sure if the move they made had any impact here

**Comment by carver** [comment]
This is blocked on upgrading jsonrpsee. It comes with a handful of breaking changes.

**Comment by carver** [comment]
Yeah, you're right, it wasn't the move. It was because their versioning went backwards and into a pre-release. The linked PR will resolve the issue.
